mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -4075,9 +4079,64 @@ resource "google_compute_instance" "foobar" {
   network_interface {
     network = "default"
   }
+}
+`, context)
+}
 
-  metadata = {
-    foo = "bar"
+func testAccComputeInstance_resourceManagerTagsUpdate(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_tags_tag_key" "key" {
+  parent = "projects/%{project}"
+  short_name = "foobarbaz%{random_suffix}"
+  description = "For foo/bar resources."
+}
+
+resource "google_tags_tag_value" "value" {
+  parent = "tagKeys/${google_tags_tag_key.key.name}"
+  short_name = "foo%{random_suffix}"
+  description = "For foo resources."
+}
+
+resource "google_tags_tag_key" "key_new" {
+  parent = "projects/%{project}"
+  short_name = "foobarbaznew%{random_suffix}"
+  description = "New key for foo/bar resources."
+}
+
+resource "google_tags_tag_value" "value_new" {
+  parent = "tagKeys/${google_tags_tag_key.key_new.name}"
+  short_name = "foonew%{random_suffix}"
+  description = "New value for foo resources."
+}
+
+data "google_compute_image" "my_image" {
+  family  = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_instance" "foobar" {
+  name           = "%{instance_name}"
+  machine_type   = "e2-medium"
+  zone           = "us-central1-a"
+
+  boot_disk {
+    initialize_params {
+      image = data.google_compute_image.my_image.self_link
+      resource_manager_tags = {
+        "tagKeys/${google_tags_tag_key.key.name}" = "tagValues/${google_tags_tag_value.value.name}"
+      }
+    }
+  }
+
+  params {
+    resource_manager_tags = {
```

Comment:
Is there any way to check that the update succeeded? I don't see this field coming back from the API response, so we can't track that the tags are what we think they are.

We may just have to trust the API that the update succeeded, but it's not ideal